iphone/cli/commands/_build.js

```diff
@@ -1899,7 +1909,7 @@ iOSBuilder.prototype.validate = function validate(logger, config, cli) {
 			// make sure they have Apple's WWDR cert installed
 			if (!this.iosInfo.certs.wwdr) {
 				logger.error(__('WWDR Intermediate Certificate not found') + '\n');
-				logger.log(__('Download and install the certificate from %s', 'https://www.apple.com/certificateauthority/AppleWWDRCAG3.cer'.cyan) + '\n');
+				logger.log(__('Download and install the certificate from %s', 'https://www.apple.com/certificateauthority/AppleWWDRCAG2.cer'.cyan) + '\n');
```

Comment:
careful: we had people that complaint that 2 is not working and only 3 worked for them: https://github.com/tidev/titanium-sdk/pull/13841 We should keep 3 here

Reply:
ah, sorry, my mistake (when using older code) :-(